### PR TITLE
fix: change default coin after execute setup.sh

### PR DESCRIPTION
**Pull request by 170210**
## Description
This PR fix the base coin error after executing ```setup.sh``` script.

### CODE REVIEW

**Review by jaeseung-bae — DISMISSED** [review]
LGTM

**Review by zemyblue — COMMENTED** [review]
Could you change also the setup.sh scripts under `with-ibc`?

**Comment by 170210** [comment]
I found that ```setup.sh``` scripts under ```with-ibc``` use two coins, including stake. So I will verify whether the script can work without any modifications.

File: localnet/single/template/setup.sh [hunk 1]
```diff
@@ -12,6 +12,7 @@ CHAIN_ID="simd-testing"
 MONIKER="simd-testing"
 CONFIG_DIR=${SCRIPT_DIR}/.finschia
 CHAIN_DIR=${CONFIG_DIR}
+DEMON="cony"
```
Comment (jaeseung-bae) [inline localnet/single/template/setup.sh]: Oh It'is typo, isn't it? 
Did you intend to type DENOM?

Comment (170210) [reply localnet/single/template/setup.sh]: Oh, that's my typo mistake, I'll fix it. Thank you very much.

**Comment by 170210** [comment]
@zemyblue After confirmation, there are no issues after running ```setup.sh``` in the ```with-ibc``` directory. finschia and finschia2 can be successfully launched. This is because both finschia and finschia2 in ```with-ibc``` have the stake denom, while ```single``` only has cony.

**Review by jaeseung-bae — APPROVED** [review]

**Review by tkxkd0159 — APPROVED** [review]

**Review by zemyblue — APPROVED** [review]

**Pull request merged**